Server/ConcordiaCurriculumManager/Repositories/CourseRepository.cs

```diff
@@ -36,14 +35,7 @@ public async Task<bool> SaveCourse(Course course)
         await _dbContext.Courses.AddAsync(course);
         var result = await _dbContext.SaveChangesAsync();
         return result > 0;
-    }
-
-    public async Task<bool> UpdateCourse(Course course)
-    {
-        _dbContext.Courses.Update(course);
-        var result = await _dbContext.SaveChangesAsync();
-        return result > 0;
     }
 
-    public async Task<Course?> GetCourseByGuid(Guid id) => await _dbContext.Courses.Where(course => course.Id == id).FirstOrDefaultAsync();
+    public async Task<Course?> GetCourseByCourseId(int CourseId) => await _dbContext.Courses.Where(course => course.CourseID == CourseId).FirstOrDefaultAsync();
```

Comment:
Can you modify this query to search for courses with CourseState = accepted (you shouldnt be able to modify non-published courses)

Reply:
Done